exercises/practice/pangram/.approaches/all-find/content.md

```diff
@@ -0,0 +1,83 @@
+# `all` with `find`
+
+```d
+module pangram;
+
+import std.algorithm.searching : all, find;
+import std.range : empty;
+import std.uni : toLower;
+
+private immutable abc = "abcdefghijklmnopqrstuvwxyz";
```

Comment:
Could this also be a `const`? I'm not particularly well-versed in D though, so feel free to ignore

Reply:
@ErikSchierboom:  D has a bit of a quirk between immutable and const.

>Since immutable data does not change, it enables many opportunities for program optimization, and has applications in functional style programming. const applies to data that cannot be changed by the const reference to that data. It may, however, be changed by another reference to that same data.

It's kind of like Ruby, it's constant but it's not constant (unless you use `freeze`.)

https://dlang.org/spec/const3.html#:~:text=Since%20immutable%20data%20does%20not,reference%20to%20that%20same%20data.

Reply:
@ErikSchierboom:  BTW, it would be good to run your solution again so that it shows up as passing the tests.

Reply:
Thanks for the immutable reference!